### PR TITLE
Update WifiManagerSnippet.java

### PR DESCRIPTION
**Pull request by kaishi05**
Add isTdlsSupported() and isAwareAvailable()

- - -
This change is [Reviewable](https://reviewable.io/reviews/google/mobly-bundled-snippets/202)

### CODE REVIEW

File: src/main/java/com/google/android/mobly/snippet/bundled/WifiManagerSnippet.java [hunk 1]
```diff
@@ -392,6 +394,25 @@ public boolean wifiIs5GHzBandSupported() {
...
+    @RequiresApi(Build.VERSION_CODES.LOLLIPOP)
+    @RpcMinSdk(Build.VERSION_CODES.LOLLIPOP)
+    @Rpc(description = "check if TDLS is supported).")
+    public boolean isTdlsSupported() {
+        return mWifiManager.isTdlsSupported();
+    }
+
+     /** Checks if Aware is available. This could return false if WiFi or location is disabled. */
+    @Rpc(description = "check if Aware is available.")
+    public boolean isAwareAvailable() throws Throwable {
```
Comment (xpconanfan) [inline src/main/java/com/google/android/mobly/snippet/bundled/WifiManagerSnippet.java:407]: this class is not the place to add wifi aware functionalities.

pls remove

**Review by xpconanfan — CHANGES_REQUESTED** [review]

**Comment by xpconanfan** [comment]
please provide contextual PR title and descriptions
current ones are too generic

File: src/main/java/com/google/android/mobly/snippet/bundled/WifiManagerSnippet.java (hunk 1 above)
Comment (kaishi05) [reply src/main/java/com/google/android/mobly/snippet/bundled/WifiManagerSnippet.java]: Move Aware change to a new snippet file.

**Review by xpconanfan — APPROVED** [review]

**Comment by kaishi05** [comment]
Please help merge the change.

**Pull request merged**